src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -119,62 +119,52 @@ rpmostree_sysroot_upgrader_initable_init (GInitable        *initable,
                                           GCancellable     *cancellable,
                                           GError          **error)
 {
-  gboolean ret = FALSE;
   RpmOstreeSysrootUpgrader *self = (RpmOstreeSysrootUpgrader*)initable;
+
   OstreeDeployment *booted_deployment =
     ostree_sysroot_get_booted_deployment (self->sysroot);
-  const char *merge_deployment_csum = NULL;
-  gboolean is_layered;
-
   if (booted_deployment == NULL && self->osname == NULL)
-    {
-      g_set_error_literal (error, G_IO_ERROR, G_IO_ERROR_FAILED,
-                           "Not currently booted into an OSTree system and no OS specified");
-      goto out;
-    }
+    return glnx_throw (error, "Not currently booted into an OSTree system and no OS specified");
```

Comment:
I'm starting to come around on my hard 80 char limit at least for error message strings like these because my need to be able to easily read files side by side is trumped by the ability to easily grep error messages.